core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-page-dropzone/ema-page-dropzone.component.ts

```diff
@@ -30,12 +37,7 @@ export class EmaPageDropzoneComponent {
     @Input() item: EmaDragItem;
     @Output() place = new EventEmitter<PositionPayload>();
 
-    pointerPosition: Record<string, string> = {
-        left: '0',
-        width: '0',
-        opacity: '0',
-        top: '0'
-    };
+    pointerPosition: Record<string, string> = POINTER_INITIAL_POSITION;
```

Comment:
Is this read-only?

Reply:
No, we change the value of it depending on the state of this component